### ISSUE TITLE
Write automated tests: Test folder missing

### ISSUE THREAD

**Issue by vanchess-git** [google-developer-training/basic-android-kotlin-compose-training-tip-calculator#56]
the given project files download does not contain the test directory in "project_name"/app/src. making it impossible to follow the instruction of part 4. Write a local test 
1. In the Project tab, click app > test and then click the d4706c21930a1ef3.png expander arrow next to the test directory.

**Comment by GoliathEng** [comment]
I noticed that too. I was following [ Write automated tests ] > (5) Write an instrumentation test, where instead of **Tests Passed**, I got BUILD FAILED. Therefore I downloaded the project file. Upon launching the downloaded project, I got the Gradle project sync failed. The Build message says 

"The project is using an incompatible version (AGP 7.3.1) of the Android Gradle plugin. Latest supported version is AGP 7.2.1
See Android Studio & AGP compatibility options."

After that I switch to Project > [basic-android-kotlin-compose-training-tip-calculator-main] > [app] > [src]
Under [src] directory there is only one [main]. 
There is no [androidTest], hence I also agree that it not possible to follow.
